unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh

```diff
@@ -168,35 +168,40 @@ User:
     fi
 
     if [ -n "${CHANGEALL}" ]; then
-        for i in "${!USERS[@]}"; do
```

Comment:
The changes in these lines correspond to a fix for https://github.com/wazuh/wazuh-packages/pull/937 that slipped through